src/app-bridge/app-bridge.ts

```diff
@@ -68,23 +69,30 @@ const createEmptySubscribeMap = (): SubscribeMap => ({
 
 export type AppBridgeOptions = {
   targetDomain?: string;
+  saleorApiUrl?: string;
   initialLocale?: LocaleCode;
 };
 
 /**
  * TODO: Consider validating locale if wrong code provided
  */
 const getLocaleFromUrl = () =>
-  (new URL(window.location.href).searchParams.get("locale") as LocaleCode) || undefined;
+  (new URL(window.location.href).searchParams.get(AppIframeParams.LOCALE) as LocaleCode) ||
+  undefined;
 
 /**
  * TODO: Probably remove empty string fallback
  */
-const getDomainFromUrl = () => new URL(window.location.href).searchParams.get("domain") || "";
+const getDomainFromUrl = () =>
+  new URL(window.location.href).searchParams.get(AppIframeParams.DOMAIN) || "";
+
+const getSaleorApiUrlFromUrl = () =>
+  new URL(window.location.href).searchParams.get(AppIframeParams.SALEOR_API_URL) || "";
```

Comment:
Is there a need to provide both `?domain=…` and `?saleorApiUrl=…`? I think we should not require the `domain` param as it can be generated from `saleorApiUrl`.

Reply:
This will still error if only `saleorApiUrl` is provided:
https://github.com/saleor/saleor-app-sdk/blob/5b9e4bb2f5f068c8aa86f56d5375f78d17e72961/src/app-bridge/app-bridge.ts#L132-L136

Reply:
We can, but I wanted to make an addition, without breaking existing flows, because I'm not sure about the effort of existing apps.

My goal was to add extra param, on top of existing "domain", so new apps can use it now, old ones can slowly migrate 

Requiring "?domain" is contact between App and Dashboard only

I suggest that we make an addition, start using it and then create a migration guide

Reply:
My suggestion would not break existing apps but allow new apps only to use `saleorApiUrl` which I believe is the end goal.

Reply:
I can do that. But technically for app it doesn't change, because its dashboard which provides both. App will just have one more field

Reply:
Done, now error is printed only if both params are missing